tests/tests_unit/test_cli_validate.py

```diff
@@ -120,9 +120,10 @@ def test_validate_error(config_yaml: str, expected_message: str):
     assert "ERROR validating configuration" in result.stdout
     if expected_message:
         # since this will usually come from a parsed filename, assume spacing/hyphenation/case is optional
-        assert (expected_message in result.stdout.lower()) or (
+        actual_message = result.stdout.lower().replace("\n", "")
+        assert (expected_message in actual_message) or (
             expected_message.replace("-", " ").replace("_", " ")
-            in result.stdout.lower()
+            in actual_message
         )
```

Comment:
xref #1938 
This assertion will fail without the `-s` flag while running pytest.

In the following case, there is a `\n` in the actual error message.
```
---------------------------
Expected message:
extra-fields

Actual message:
ERROR validating configuration 
/Users/fangchenli/Workspace/nebari/tests/tests_unit/cli_validate/**_local.error.ext
ra-fields.yaml_**
1 validation error for ConfigSchema
this_is_an_error
  Extra inputs are not permitted 
    For further information visit 
https://errors.pydantic.dev/2.2/v/extra_forbidden
Aborted.
```